srcpkgs/papis/template

```diff
@@ -1,22 +1,34 @@
 # Template file for 'papis'
 pkgname=papis
-version=0.11.1
-revision=2
+version=0.12
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-requests python3-yaml python3-chardet python3-BeautifulSoup4
- python3-colorama python3-click python3-slugify python3-prompt_toolkit
- python3-tqdm python3-Pygments python3-stevedore python3-parsing
- python3-filetype python3-bibtexparser python3-habanero python3-arxiv2bib
- python3-isbnlib python3 python3-setuptools python3-lxml
- python3-typing_extensions python3-doi"
+depends="python3-requests python3-filetype python3-parsing python3-arxiv2bib
+ python3-yaml python3-chardet python3-BeautifulSoup4 python3-colorama
+ python3-bibtexparser python3-click python3-habanero python3-isbnlib
+ python3-prompt_toolkit python3-tqdm python3-Pygments python3-stevedore
+ python3-doi python3-lxml python3-slugify python3-Whoosh"
```

Comment:
re-ordering all of these dependencies makes for a hard-to-parse diff, and shouldn't be necessary

Reply:
I was following the ordering in https://github.com/papis/papis/blob/v0.12/setup.py#L55 cause there are so many in case I miss something or something redundant gets into it.

Reply:
I'd prefer not changing the order, it makes the history easier to use